scripts/deployment-helpers.sh

```
@@ -739,13 +1185,21 @@ wait_authorino_ready() {
     echo "  WARNING: Could not determine gateway URL, skipping request verification"
     return 0
   fi
+
+  echo "  - Using gateway URL: $maas_url"
+  if [[ -z "$maas_url" ]]; then
+    echo "  WARNING: Could not determine gateway URL, skipping request verification"
+    return 0
+  fi
+
+  echo "  - Using gateway URL: $maas_url"
+>>>>>>> ea56772 (rebase PR and include changes from 381)
```

Shellcheck (0.11.0)

[error] Couldn't parse this redirection. Fix to allow more checks.

(SC1073)